tests-utils/src/main/resources/ldap.ldif

```diff
@@ -1,230 +0,0 @@
-#################################
```

Comment:
I guess it wasn't used?

Reply:
Yes, `integrations-tests` had its own version, same as `core` tests.
This PR changed the way resources are handled - we now reuse resources from tests-utils for integration-tests and ror-utils tests. This file was unused anyway and started causing conflicts.